version/patches.go

```diff
@@ -0,0 +1,12 @@
+package version
+
+/*
+	For each patch (a.k.a. class3 upgrade), define the corresponding height
+	at which the software switches to the patched logic
+*/
+const (
+	//	001_PATCH_HEIGHT = 15300	// 001: 0 - protocol version, 01 - patch number for protocol v0
+	//	101_PATCH_HEIGHT = 26000
+	//	102_PATCH_HEIGHT = 75000
+	P001_UNDELEGATE_HEIGHT = 75000
```

Comment:
Is the height correct?

Reply:
The correct height, once determined, will be updated. @HaoyangLiu